client/src/components/App.js

```diff
@@ -41,6 +42,7 @@ const App = (props) => {
         <Route exact path="/users/new" component={RegistrationForm} />
         <Route exact path="/user-sessions/new" component={SignInForm} />
         <AuthenticatedRoute exact path="/profile" component={UserProfile} user={currentUser} />
+        <AuthenticatedRoute exact path="/new" component={NewParkForm} user={currentUser} />
```

Comment:
Hey y'all! I would actually argue that this path SHOULD be `/parks/new` in order to stick to RESTful conventions. Can you tell me a little more about why `/parks/new` was giving you errors?

Reply:
So for some reason when we posted to /parks/new it was also hitting the parksRouter.get("/:id"... and was sending back a 500 error saying it couldn't fetch a park with id "new".  As I'm typing it I'm sure there's a reason why it was hitting the get request and that should probably be figured out...

Reply:
Ahhhh yes 😄  this is a tricky one. Dynamic routes can often cause this issue. Order matters here!

If you actually move this route (line 45) ABOVE the dynamic route (line 41), that should fix it for ya :)